test/.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "parserOptions": {
+    "ecmaVersion": 6
```

Comment:
left a note there, too. I don't see why the ES6 rewrite was necessary. Stateless components are simply functions with signature `function(props) { return whateveryouneedrendered }`.